content/doc/book/installing/index.adoc

```diff
@@ -238,6 +238,50 @@ If your `/etc/init.d/jenkins` file fails to start Jenkins, edit the `/etc/defaul
 Here, "8081" was chosen but you can put another port available.
 ====
 
+===== Installation of Java
+
+Jenkins requires Java in order to run, yet certain distributions don't include this by default and  link:https://www.jenkins.io/doc/administration/requirements/java/[some Java version are incompatible] with jenkins.
+
+To install the Open Java Development Kit (OpenJDK) run the following:
+
+* Update the repositories 
+[source,bash]
+----
+sudo apt update
+----
+* search of all available packages:
+[source,bash]
+----
+sudo apt search openjdk
+----
+* Pick one option and install it: 
+[source,bash]
+----
+sudo apt install openjdk-8-jdk
+----
+* Confirm installation:
+[source,bash]
+----
+sudo apt install openjdk-8-jdk 
+----
+* checking installation:
+[source,bash]
+----
+java -version
+----
+* the result must be something like:
+[source,bash]
+----
+openjdk version "1.8.0_252"
+OpenJDK Runtime Environment (build 1.8.0_252-8u252-b09-1ubuntu1-b09)
+OpenJDK 64-Bit Server VM (build 25.252-b09, mixed mode)
+----
+
+[NOTE]
```

Comment:
Not sure we should retain it in the guide. This comment is rather a personal opinion from somebody, and it is not that relevant to the guide itself

Reply:
Hey @oleg-nenashev, thanks for your time. 
I'd added the note because it was required after the first review. It isn't a personal opnion. I tried to use the explanation from the official site  [manpages.ubuntu](http://manpages.ubuntu.com/manpages/trusty/man8/apt.8.html#differences%20to%20apt-get(8)) please check the link.

So what should I do with te note section? Keep it? change it? Remove it? 

I'm waitning for your answer.

Reply:
Let's keep it then since it was recommended. I do not have a strong opinion here